.travis.yml

```diff
@@ -126,11 +128,10 @@ before_script:
 # Tests running commands
 script:
   # make frontend_install_tests takes 17 min, so we only do it post submit
-  - "[ $TESTS != js -o ${TRAVIS_PULL_REQUEST} = false ] || make frontend"
-  - "[ $TESTS != js -o ${TRAVIS_PULL_REQUEST} != false ] || make frontend_install_tests"
+  - "if [ $TESTS == js -a ${TRAVIS_PULL_REQUEST} == false ]; then make frontend ; fi"
+  - "if [ $TESTS == js -a ${TRAVIS_PULL_REQUEST} == false ]; then make frontend_install_tests ; fi"
 
   - |
-    TESTS=py3status
```

Comment:
Yes, this is a fix.  You aren't supposed to assign TESTS=pystatus inside the script.
It needs to be done in the matrix section.